e2e/actions/oci.go

```diff
@@ -1236,8 +1243,66 @@ func (c actionTests) actionOciOverlay(t *testing.T) {
 					"--overlay", filepath.Join(testDir, "my_rw_ol_dir2"),
 					imageRef, "cat", "/testfile.1", "/maskable_testfile", filepath.Join("/", imgTestFilePath),
 				},
-				requiredCmds: []string{"squashfuse"},
-				exitCode:     255,
+				exitCode: 255,
+			},
+			{
+				name:         "WritableExtfs",
+				args:         []string{"--overlay", writableExtfsImgPath, imageRef, "sh", "-c", "echo my_test_string > /my_test_file"},
+				requiredCmds: []string{"fuse2fs", "fuse-overlayfs", "fusermount"},
+				exitCode:     0,
+			},
+			{
+				name:         "WritableExtfsRevisit",
```

Comment:
We need to revisit the `rw` overlay in `ro` mode too, so we catch issue (2) in my general comments.